Author: Danny Milosavljevic <dannym@friendly-machines.com>
Date: 2026-02-08
License: ASL2.0

Remove [patch.crates-io] and [patch.'ssh://...'] sections.

These sections point to git repositories which are unavailable in an
offline build.  The vendored tokio-tungstenite and tungstenite forks
are provided as cargo-inputs instead.

diff -u a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,8 +47,2 @@
 unused = "warn"
-
-[patch.crates-io]
-tokio-tungstenite = { git = "https://github.com/JakkuSakura/tokio-tungstenite", branch = "proxy-support" }
-
-[patch.'ssh://git@github.com/JakkuSakura/tungstenite-rs.git']
-tungstenite = { git = "https://github.com/JakkuSakura/tungstenite-rs", branch = "proxy-support" }
